### ISSUE TITLE
Recording starting one hour too early - Time zone issue ?

### ISSUE THREAD

**Issue by gvdvg** [rytsikau/ee.Yrewind#52]
Hey, While trying to record a segment from sky news livestream (9Auq9mYxFEE), it records from one hour too early.  To get the right time I have to specify the wrong time (+1 hr). Therefore I can't record a segment from within the last hour as it's a time in the future essentially.  Also -start=-20 record from 80 minutes previous

I'm using the latest release, my system is in GMT . Maybe summer daylight savings is wrongly being accounted for somewhere ?
thanks

**Comment by rytsikau** [comment]
No, there is no manipulation of daylight saving time. Check your time zone

**Comment by archfan7411** [comment]
We've also had this issue, despite the correct timezone on the computer running it. We've resorted to storing offsets for each stream and adjusting times to get the right capture.
